File: src/main/java/io/github/vehkiya/data/model/json/JsonItem.java
package io.github.vehkiya.data.model.json;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true, fluent = true)
@JsonIgnoreProperties(ignoreUnknown = true)
public class JsonItem {

    @JsonProperty("slug")
    private String slug;

    @JsonProperty("name")
    private String name;
}

File: src/main/java/io/github/vehkiya/data/model/json/JsonItem.kt
package io.github.vehkiya.data.model.json

import com.fasterxml.jackson.annotation.JsonAutoDetect
import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
data class JsonItem(
    @JsonProperty("slug") val slug: String,
    @JsonProperty("name") val name: String
)
